erpnext/patches/v13_0/set_values_in_pick_list.py

```diff
@@ -0,0 +1,15 @@
+import frappe
+def execute():
+	frappe.reload_doctype("Pick List")
+	pick_lists= frappe.get_all("Pick List",filters= {
+		"purpose": "Delivery"
+	}, fields= ["name","customer"])
+	for pick_list in pick_lists:
+		order_delivery_date = []
+		sales_order = frappe.get_value("Pick List Item",{"parent": pick_list.name}, "sales_order")
+		if sales_order:
+			delivery_dates = frappe.get_all("Sales Order Item", {"parent": sales_order}, "delivery_date")
+			for delivery in delivery_dates:
+				order_delivery_date.append(frappe.db.get_value("Sales Order Item", {"parent": sales_order}, "delivery_date"))
+			frappe.db.set_value("Pick List", pick_list.name, "delivery_date", min(order_delivery_date), update_modified=False)
```

Comment:
Only do this if `order_delivery_date` is not an empty list

Reply:
order_delivery_date will always empty for new pick list.
![image](https://user-images.githubusercontent.com/56826544/91045060-e3ee7e80-e633-11ea-8ff7-0752e322ef69.png)

Reply:
No, I mean line 14, only do a `set_value` if `order_delivery_date` is not empty, otherwise `min()` will fail for an empty sequence

Reply:
@nmimsnikhil, can you look at this one?

Reply:
ya i have made changes

Reply:
![image](https://user-images.githubusercontent.com/56826544/91162032-2756f480-e6e9-11ea-895c-df6f3e71ba5e.png)

Reply:
Oh, I missed it in the diff, this is fine, I'm just testing it out now